pkg/workceptor/kubernetes.go

```diff
@@ -39,6 +39,7 @@ import (
 // kubeUnit implements the WorkUnit interface.
 type kubeUnit struct {
 	BaseWorkUnit
+	// BaseWorkUnitForWorkUnit
```

Comment:
So this is just implemented for the command work unit and kube is next?

Reply:
Yes kube is next and remote work. This interface will work for all the work types. I can swap BaseWorkUnit for BaseWorkUnitForWorkUnit in this PR

Reply:
Well it looks like I already did remote_work. I can do kube as well. Wdyt?

Reply:
Im easy, both are good options. Perhaps Kube after will help with your sanity :grin: